web/src/auth.ts

```diff
@@ -1,10 +1,13 @@
-export const authConfig = {
+import { TAuthConfig } from 'react-oauth2-code-pkce'
+
+export const authConfig: TAuthConfig = {
   clientId: process.env.REACT_APP_AUTH_CLIENT_ID || '',
   authorizationEndpoint: process.env.REACT_APP_AUTH_ENDPOINT || '',
   tokenEndpoint: process.env.REACT_APP_TOKEN_ENDPOINT || '',
   scope: process.env.REACT_APP_AUTH_SCOPE || '',
   redirectUri: process.env.REACT_APP_AUTH_REDIRECT_URI || '',
   logoutEndpoint: process.env.REACT_APP_LOGOUT_ENDPOINT || '',
+  autoLogin: false,
```

Comment:
If y'all thing we should re-enable `autoLogin`, I will.

Reply:
roger